### ISSUE TITLE
No transaction handling

### ISSUE THREAD

**Issue by kunalbhatia87** [shekar21/Final-Bank#5]
Shouldn't there be transaction handling for rollback in case of exceptions

**Comment by shekar21** [comment]
Yes even i thought of to have it. Assumed the template takes care of it. :-).. Will check and add..

**Comment by shekar21** [comment]
Done transaction management
